test/unittests/instantiate_test.cpp

```diff
@@ -56,6 +56,17 @@ TEST(instantiate, imported_functions_not_enough)
         "Module requires 1 imported functions, 0 provided");
 }
 
+TEST(instantiate, imported_function_wrong_type)
+{
+    Module module;
+    module.typesec.emplace_back(FuncType{{ValType::i32}, {ValType::i32}});
+    module.importsec.emplace_back(Import{"mod", "foo", ExternalKind::Function, {0}});
```

Comment:
```suggestion
    module.importsec.emplace_back(Import{"mod", "foo", ExternalKind::Function, {FuncIdx{0}}});
```

Reply:
This repeats the pattern in other tests above, should I change it there, too?

Reply:
Yeah, sure.

Reply:
changed